cfgov/unprocessed/css/pages/home.less

```diff
@@ -15,3 +15,20 @@
         .u-info-unit-base__inline( @m-info-unit_img__lg );
     } );
 }
+
+.o-well__home {
+    .respond-to-min(@bp-sm-min, {
+        .content-l__large-gutters {
```

Comment:
I think it could -- I'll update after moving the equal height classes to layout.